open_academy/models/course.py

```diff
@@ -10,3 +10,21 @@ class Course(models.Model):
     description = fields.Text()
     responsible_user_id = fields.Many2one('res.users')
     session_ids = fields.One2many('session', 'course_id')
+    _sql_constraints = [
+        ('unique_course_name', 'UNIQUE (name)', 'This course already exists'),
+        (
+            'check_title_description',
+            'CHECK (title NOT LIKE description)',
+            'The title must be different than the description'
+        )
+    ]
+
+    def copy(self, default=None):
+        default = dict(default or {})
```

Comment:
<img width="395" alt="Screen Shot 2022-08-29 at 7 36 17 p m" src="https://user-images.githubusercontent.com/47955967/187322531-5561f8ab-2efe-436d-b8b4-0c45ba69aa6c.png">

```suggestion
        default = default or {}
```